packages/components/care-card/README.md

```diff
@@ -107,9 +107,9 @@ If you’re using Nunjucks macros in production be aware that using `html` argum
 ### HTML markup
 
 ```html
-<div class="nhsuk-care-card nhsuk-care-card--immediate">
+<div class="nhsuk-care-card nhsuk-care-card--immediate" >
```

Comment:
Would you be able to look at the care-card njk file remove the empty space between the `"` and closing tag `>` and also remove it from here?